seeds/addresses_events/first_funded_by_seed_bitcoin.csv

```diff
@@ -0,0 +1,3 @@
+blockchain,address,first_funded_by,block_height,tx_id
+bitcoin,1LgPGjK1v4DjoXnkAUHDKcPcnXbj1cVZN3,"[""1DQqGWgWH44gpkQcTZobK4nymtTZRo1ey3"", ""1KKqRoee3msNxfQUj1yXGbboL8aNiNuXS5""]",72984,0xe4941ec79d6a3c8c9f9fa3d0e63071ecbe2d40f80b8849bf3c58f805a4898dfc
+bitcoin,18cQigCXf6ijgstj1G5LX1SW7b5ktB41Vb,"[""1DACGAnLLgki4QYwX59hCrTAj7Aczz57JR"", ""1NWwckJZvV4st7iDdvfed6CvnZrKrTXrT9""]",75870,0xbb5d419486a9bd58f9eece840ef17bcdeffe05a8682213dd2481d11b53fdc79a
```

Comment:
If autoparsing still fail, we may need to mannually write the test with `json_parse()` function, like:
```
SELECT
  blockchain,
  address,
  json_parse(first_funded_by) AS first_funded_by_array,
  block_height,
  tx_id
FROM {{ ref('first_funded_by_seed_bitcoin') }}
```

```suggestion
blockchain,address,first_funded_by,block_height,tx_id
bitcoin,1LgPGjK1v4DjoXnkAUHDKcPcnXbj1cVZN3,'["1DQqGWgWH44gpkQcTZobK4nymtTZRo1ey3", "1KKqRoee3msNxfQUj1yXGbboL8aNiNuXS5"]',72984,0xe4941ec79d6a3c8c9f9fa3d0e63071ecbe2d40f80b8849bf3c58f805a4898dfc
bitcoin,18cQigCXf6ijgstj1G5LX1SW7b5ktB41Vb,'["1DACGAnLLgki4QYwX59hCrTAj7Aczz57JR", "1NWwckJZvV4st7iDdvfed6CvnZrKrTXrT9"]',75870,0xbb5d419486a9bd58f9eece840ef17bcdeffe05a8682213dd2481d11b53fdc79a
```

Reply:
> If autoparsing still fail, we may need to mannually write the test with `json_parse()` function, like:
> 
> ```
> SELECT
>   blockchain,
>   address,
>   json_parse(first_funded_by) AS first_funded_by_array,
>   block_height,
>   tx_id
> FROM {{ ref('first_funded_by_seed_bitcoin') }}
> ```

ideally there's a way to parse arrays in test but maybe there isn't. @jeff-dude @0xRobin have you guys dealt with this before?

if not we can do the manual test instead but i'm not sure how those work, where would you place that query then @Hosuke?

Reply:
ARRAY type in seeds is not supported in dbt I'm afraid. 
string -> json is the only workaround but does require a custom test.

Reply:
ah ok. i'll just remove that column from being checked then